server/protocols/jmap-rfc-8621/src/test/scala/org/apache/james/jmap/routes/JMAPApiRoutesTest.scala

```diff
@@ -303,7 +331,47 @@ class JMAPApiRoutesTest extends AnyFlatSpec with BeforeAndAfter with Matchers {
         .body(WRONG_OBJECT_REQUEST)
       .when()
         .post
-      .then
+      .`then`
+        .statusCode(HttpStatus.SC_BAD_REQUEST)
+        .body("status", equalTo(400))
+        .body("type", equalTo(RequestLevelErrorType.NOT_REQUEST.value))
+        .body("detail", equalTo("The request parsed as JSON but did not match the type signature of the Request object: Invalid RequestObject: List((/methodCalls,List(JsonValidationError(List(error.expected.jsarray),ArraySeq()))))"))
+  }
+
+  "RFC-8621 version, POST, with not json request body" should "return 400 status" in {
+    val headers: Headers = Headers.headers(
+      new Header(ACCEPT.toString, ACCEPT_RFC8621_VERSION_HEADER),
+      new Header("Authorization", s"Basic ${userBase64String}")
+    )
+    RestAssured
+      .`given`()
+        .headers(headers)
+        .body(NOT_JSON_REQUEST)
+      .when()
+        .post
+      .`then`
+        .statusCode(HttpStatus.SC_BAD_REQUEST)
+        .body("status", equalTo(400))
+        .body("type", equalTo(RequestLevelErrorType.NOT_JSON.value))
+        .body("detail", containsString("The content type of the request was not application/json or the request did not parse as I-JSON"))
+        .body("detail", containsString("Unexpected character ('}' (code 125)): was expecting double-quote to start field name"))
+  }
+
+  "RFC-8621 version, POST, with unknown capability" should "return 400 status" in {
+    val headers: Headers = Headers.headers(
+      new Header(ACCEPT.toString, ACCEPT_RFC8621_VERSION_HEADER),
+      new Header("Authorization", s"Basic ${userBase64String}")
+    )
+    RestAssured
+      .`given`()
+        .headers(headers)
+        .body(UNKNOWN_CAPABILITY_REQUEST)
+      .when()
+        .post
+      .`then`
         .statusCode(HttpStatus.SC_BAD_REQUEST)
+        .body("status", equalTo(400))
+        .body("type", equalTo(RequestLevelErrorType.UNKNOWN_CAPABILITY.value))
+        .body("detail", equalTo("The request used unsupported capabilities: List(urn:ietf:params:jmap:core1)"))
```

Comment:
just a little advice for next time: you can commit each test with the matching behavior, making this more friendly for reviewers